setuptools/msvc.py

```diff
@@ -212,6 +221,19 @@ def msvc14_get_vc_env(plat_spec):
         raise
 
 
+def msvc14_gen_lib_options(*args, **kwargs):
+    """
+    Patched "distutils._msvccompiler.gen_lib_options" for fix 
+    compatibility between "numpy.distutils" and "distutils._msvccompiler"
+    (for Numpy < 1.11.2)
+    """
+    if "numpy.distutils" in sys.modules:
+        import numpy as np
+        if StrictVersion(np.__version__) < StrictVersion('1.11.2'):
```

Comment:
I dont find `pkg_resources.extern.packaging.version`. But `pkg_resources.parse_version`, if you prefer

```
In [20]: pkg_resources.parse_version('1.11.1rc1+fa248ad') < pkg_resources.parse_version('1.11.2')
Out[20]: True
In [21]: pkg_resources.parse_version('1.11.3rc1+fa248ad') < pkg_resources.parse_version('1.11.2')
Out[21]: False
```